spek-runtimes/jvm/src/main/kotlin/org/spekframework/spek2/runtime/SpekJvmRuntime.kt

```diff
@@ -36,17 +36,14 @@ actual class SpekRuntime: AbstractRuntime() {
             .map(Class<out Spek>::kotlin)
             .filter { it.findAnnotation<Ignore>() == null }
             .filter { !it.isAbstract }
-            .filter {
-                val path = PathBuilder.from(it)
-                    .build()
-
+            .map { klass ->
+                klass to JvmPathBuilder.from(klass).build()
```

Comment:
```console
Unresolved reference: JvmPathBuilder
```

There is no more `JvmPathBuilder`, it's now just `PathBuilder` implemented for every platform :)